src/ProgressiveCurve.sol

```diff
@@ -165,7 +167,7 @@ contract ProgressiveCurve is BaseCurve {
     /// @dev And the slope ($m$) determines how quickly the price increases
     /// @dev TLDR: Each new share costs more than the last
     function currentPrice(uint256 totalShares) public view override returns (uint256 sharePrice) {
-        return UD60x18.wrap(totalShares).mul(SLOPE).unwrap();
```

Comment:
3.6 - currentSharePrice and currentSharePriceCurve Return Price With Orders of Magnitude Inflated by generalConfig.decimalPrecision

We addressed this by updating the comments and ensuring that current share price is always represented by a factor of 1e18, for both pro-rata vaults and curve vaults.